allofplos/corpus/plos_corpus.py

```diff
@@ -278,9 +278,16 @@ def compare_article_pubdate(doi, days=22, directory=None):
         print("Pubdate error in {}".format(doi))
 
 
+def download_xml(doi, tempdir=newarticledir):
```

Comment:
That is really clever! I hadn't thought of using an article object as a downloader… why don't we just bake this into the article class itself?